tests/trainer/test_optimizers.py

```diff
@@ -309,25 +346,68 @@ def test_configure_optimizers_with_frequency(tmpdir):
     model = EvalModelTemplate()
     model.configure_optimizers = model.configure_optimizers__multiple_optimizers_frequency
 
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1
-    )
+    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1)
+    result = trainer.fit(model)
+    assert result
+
+
+def test_configure_optimizers_with_frequency_and_lr_schedulers(tmpdir):
+    """
+    Test that opt_idx is set for LR schedulers when corresponding frequency is set for multiple optimizers.
+    """
+    model = EvalModelTemplate()
```

Comment:
Could you please use BoringModel instead ?

Reply:
Yes I can do that! Should I use `BoringModel` for all newly added tests?

Reply:
yes, pls :]